Add documentation about `job` parameter

After upgrade to 2.5.0, I get the following error
```
[ERROR] Failed to execute goal com.exasol:project-keeper-maven-plugin:4.5.0:fix (default-cli) on project cloud-storage-extension: Execution default-cli of goal com.exasol:project-keeper-maven-plugin:4.5.0:fix failed: E-PK-CORE-208: Missing job in step customization of workflow 'ci-build.yml' in file '.project-keeper.yml'. Add job with one of the following values: ['matrix-build', 'next-java-compatibility', 'build', 'start_release']. -> [Help 1]
[ERROR] 
```

Was fixed by adding `job: matrix-build` to the ci customizations parameters, but it wasn't described in documentation.
It would be great to document this.

--- a/doc/changes/changes_5.0.0.md
+++ b/doc/changes/changes_5.0.0.md
@@ -22,6 +22,8 @@ The release also moves execution of `display-plugin-updates` and `display-depend
 
 The release also improves the error message for unknown job ID in workflow customization. Users now directly get a list of available job IDs they can use instead.
 
+The release also adds documentation of `job` parameter for workflow customizations to the user guide.
+
 ## Features
 
 * #609: Adjusted Eclipse formatter to never join wrapped lines and to indent `case` inside of `switch`
@@ -38,6 +40,7 @@ The release also improves the error message for unknown job ID in workflow custo
 
 * Added security policy (PR #618)
 * Added clarifications for custom artifacts (PR #621)
+* #610: Added documentation of `job` field for workflow customizations
 
 ## Dependency Updates
 

--- a/doc/user_guide/user_guide.md
+++ b/doc/user_guide/user_guide.md
@@ -298,6 +298,7 @@ build:
     - name: "ci-build.yml"
       stepCustomizations:
         - action: REPLACE | INSERT_AFTER
+          job: <job-name>
           stepId: <step-id>
           content:
             name: <Step name>
@@ -316,6 +317,7 @@ build:
   * `action`: Type of customization
     * `REPLACE`: Replace an existing step with the new content
     * `INSERT_AFTER`: Insert the content **after** the specified step
+  * `job`: Name of the job inside the workflow that should be modified
   * `stepId`: ID of the step to replace or after which to insert the new step
   * `content`: Content of the new step. PK does not validate this. You can use any valid GitHub action using `run` or `uses`, see examples below.
 
@@ -328,6 +330,7 @@ build:
     - name: "ci-build.yml"
       stepCustomizations:
         - action: INSERT_AFTER
+          job: build-and-test
           stepId: setup-java
           content:
             name: Set up Go
@@ -337,12 +340,14 @@ build:
               go-version: "1.24"
               cache-dependency-path: .project-keeper.yml
         - action: INSERT_AFTER
+          job: build-and-test
           stepId: setup-go
           content:
             name: Install Go tools
             id: install-go-tools
             run: go install github.com/google/go-licenses@v1.6.0
         - action: REPLACE
+          job: build-and-test
           stepId: build-pk-verify
           content:
             name: Run tests and build with Maven

--- a/shared-test-setup/src/main/java/com/exasol/projectkeeper/test/NpmProjectFixture.java
+++ b/shared-test-setup/src/main/java/com/exasol/projectkeeper/test/NpmProjectFixture.java
@@ -28,13 +28,13 @@ public class NpmProjectFixture extends BaseProjectFixture {
     public void prepareProjectFiles(final Builder configBuilder) {
         this.writeConfig(configBuilder);
         this.prepareProjectFiles();
-        execute(projectDir, "npm" + suffixForWindows(".cmd"), "install");
+        execute(projectDir, "npm" + suffixForWindows(), "install");
     }
 
-    private String suffixForWindows(final String suffix) {
+    private String suffixForWindows() {
         final String os = System.getProperty("os.name", "generic").toLowerCase(Locale.ENGLISH);
         if (os.indexOf("win") >= 0) {
-            return suffix;
+            return ".cmd";
         }
         return "";
     }

